cmd/observe/observe_test.go

```diff
@@ -0,0 +1,103 @@
+package observe
+
+import (
+	"testing"
+
+	"github.com/aurae-runtime/ae/pkg/cli"
+	"github.com/aurae-runtime/ae/pkg/cli/printer"
+)
+
+func TestComplete(t *testing.T) {
+	ts := []struct {
+		args        []string
+		wantip      string
+		wantlogtype string
+		wanterr     bool
+	}{
+		{
+			[]string{""},
+			"", "", true,
+		},
+		{
+			[]string{"foo"},
+			"", "", true,
+		},
+		{
+			[]string{"foo", "bar"},
+			"foo", "bar", false,
+		},
+		{
+			[]string{"foo", "bar", "baz"},
+			"", "", true,
+		},
+	}
+
+	for _, tt := range ts {
+		o := &option{}
+		goterr := o.Complete(tt.args)
+		if tt.wanterr && goterr == nil {
+			t.Fatal("want error, got no error")
+		}
+		if !tt.wanterr && goterr != nil {
+			t.Fatal("want no error, got error")
+		}
+		if tt.wantip != o.ip {
+			t.Fatalf("want ip %q, got ip %q", tt.wantip, o.ip)
+		}
```

Comment:
Again about the port: is there a default port? From where do we infer the port?
IMO there tests should include a scenario with + without port configured.

Reply:
https://github.com/aurae-runtime/aurae/issues/347 :)